doc/Language/grammar_tutorial.pod6

```diff
@@ -27,41 +27,75 @@ language.
 
 =SUBTITLE The Broad Concept of Perl Grammars
 
-Regular Expressions (regex) work well to find patterns in strings and manipulate them. However, when you need to find multiple patterns at once, or need to combine patterns, or test for patterns that may surround strings, or other patterns - regular expressions alone are not adequate.
```

Comment:
I'd rather remove indentation at all, to be honest. It seems redundant.
About emacs... This text editor is just my own preference and it can take a long(very long) way to get used to it. If you use linux or macos, you can do a conversion by using such command:
`fold -w 80 -s filename`, `filename` must be replaced with an actual name, of course, and use the output.
Actually, I've discovered that default emacs `fill-paragraph` does formatting in 72 characters, not 80, so it is not the best option too.